lib/cloud-metadata/azure.js

```diff
@@ -38,32 +37,29 @@ function getMetadataAzure (host, port, socketTimeoutMs, httpTimeout, protocol, c
       })
 
       res.on('end', function (data) {
+        let result
         try {
-          const result = formatMetadataStringIntoObject(finalData.join(''))
-          cb(null, result)
+          result = formatMetadataStringIntoObject(finalData.join(''))
         } catch (error) {
```

Comment:
style nit: s/error/err/

Reply:
Dove tailing off the above -- is the rule we're following `err` for callbacks errors, errors handled via events, and caught exceptions?

Reply:
We could go full https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/ and Hungarian notation too! ;)

Seriously though, I try to always include "nit:" if I'm happy having it ignored.  "Rule" feels strong (I'm still finding my place), but as a proposal: yes, **"err" or "someRelevantContextNameErr" for all Error instances.**

Reply:
✅ done!